src/devices/Ft4222/Ft4222I2cBus.cs

```diff
@@ -0,0 +1,170 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Device.I2c;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Iot.Device.Ft4222
+{
+    /// <summary>
+    /// FT4222 I2C Device
+    /// </summary>
+    public class Ft4222I2cBus : IDisposable
```

Comment:
So related to the issue about I2cBus, should you directly then propose an I2cBus implementation and use FT4222 as an example of implementation?
On what I can see here, you have a good list of functions :-)

Reply:
do not want to do this as part of this PR